api/configs/middleware/__init__.py

```diff
@@ -200,5 +201,6 @@ class MiddlewareConfig(
     TencentVectorDBConfig,
     TiDBVectorConfig,
     WeaviateConfig,
+    ElasticsearchConfig
```

Comment:
```suggestion
    ElasticsearchConfig,
```

Place `ElasticsearchConfig ` after `ChromaConfig ` aligning the alphabet order.
And use a comma separator.